requirements.txt

```diff
@@ -1,3 +1,4 @@
+chardet
```

Comment:
why not cchardet?
from our experience it provides better results (and much quickly)